pkgs/development/ocaml-modules/tezos-rust-libs/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitLab, topkg, findlib, cargo, rustc, opaline }:
+
+stdenv.mkDerivation rec {
+  pname = "tezos-rust-libs";
+  version = "1.0";
+
+  src = fetchFromGitLab {
+    owner = "tezos";
+    repo = "tezos-rust-libs";
+    rev = "v${version}";
+    sha256 = "1ffkzbvb0ls4wk9205g3xh2c26cmwnl68x43gh6dm9z4xsic94v5";
+  };
+
+  buildInputs = [
+    topkg
+    findlib
+  ];
```

Comment:
Don’t you need `ocaml` there? Why did you add `topkg` and `findlib`?

Reply:
the install fails if I don't have `findlib` as a dependency, I guess it's because I'm using `$OCAMLFIND_DESTDIR` in the `installPhase`

Reply:
`topkg` can be removed, thank you